### PR TITLE
fix:clear interval fail when close autoRefresh

### PR DESCRIPTION
**Pull request by drgnchan**
fix https://github.com/apache/skywalking/issues/9224

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -101,23 +98,23 @@ const handleAuto = () => {
   appStore.setAutoRefresh(auto.value);
   if (auto.value) {
     handleReload();
-    state.timer = setInterval(handleReload, autoTime.value * 1000);
+    appStore.setReloadTimer(setInterval(handleReload, autoTime.value * 1000));
   } else {
-    if (state.timer) {
-      clearInterval(state.timer);
+    if (appStore._reloadTimer) {
```
Comment (Fine0830) [inline src/views/Settings.vue:103]: You should use `appStore.reloadTimer`.

File: src/store/modules/app.ts [hunk 2]
```diff
@@ -53,8 +54,12 @@ export const appStore = defineStore({
     pageTitle: "",
     version: "",
     isMobile: false,
+    reloadTimer: null,
   }),
   getters: {
+    _reloadTimer(): any {
```
Comment (Fine0830) [inline src/store/modules/app.ts:60]: I don't think we need this here.

File: src/views/Settings.vue [hunk 3]
```diff
@@ -101,23 +98,23 @@ const handleAuto = () => {
...
+      clearInterval(appStore._reloadTimer);
```
Comment (Fine0830) [inline src/views/Settings.vue:104]: Same problem as above.

**Review by Fine0830 — CHANGES_REQUESTED** [review]

**Comment by Superskyyy** [comment]
@drgnchan Btw next time please take the issue first before working on it, otherwise, there could be others making duplicated efforts.

File: src/store/modules/app.ts (hunk 2 above)
Comment (drgnchan) [reply src/store/modules/app.ts]: Thanks for your reminding.It was removed.

File: src/views/Settings.vue (hunk 3 above)
Comment (drgnchan) [reply src/views/Settings.vue]: done.

**Comment by drgnchan** [comment]
> @drgnchan Btw next time please take the issue first before working on it. Otherwise there could be others making duplicated efforts.

I will pay attention next time.

**Review by Fine0830 — APPROVED** [review]
LGTM. Thanks for your contribution.

**Comment by wu-sheng** [comment]
@drgnchan Once this gets merged, it is better if you could update the main repo's UI submodule with a change log to land this officially.

**Comment by drgnchan** [comment]
> @drgnchan Once this gets merged, it is better if you could update the main repo's UI submodule with a change log to land this officially.

Ok.I am not sure of this.I am going to edit file [https://github.com/apache/skywalking/blob/d5ceedbd77f344f3099ae1e17a76e2ebf937c8f1/docs/en/changes/changes.md](changes.md) and make a merge request.Am I right?

**Comment by wu-sheng** [comment]
You need to update the submodule commit ID of UI

image

If you don't understand what this means, search for `git submodule`.

**Comment by drgnchan** [comment]
> You need to update the submodule commit ID of UI
> 
> image
> 
> If you don't understand what this means, search for `git submodule`.

done.Please review the update. https://github.com/apache/skywalking/pull/9228

**Pull request merged**